specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/sqlServer.json

```diff
@@ -0,0 +1,1789 @@
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Revalidates the encryption protector": {
+            "$ref": "./examples/WorkspaceManagedSqlServerEncryptionProtectorRevalidate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlUsages": {
+      "get": {
+        "tags": [
+          "WorkspaceManagedSqlServer, sqlUsages"
+        ],
+        "operationId": "WorkspaceManagedSqlServerUsages_List",
+        "summary": "Get list of usages metric for the server.",
+        "description": "Get list of server usages metric for workspace managed sql server.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerUsageListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List usages metric for the workspace managed sql server": {
+            "$ref": "./examples/ListWorkspaceManagedSqlServerUsages.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/recoverableSqlpools": {
+      "get": {
+        "tags": [
+          "WorkspaceManagedSqlServer",
+          "recoverableSqlpools"
+        ],
+        "operationId": "WorkspaceManagedSqlServerRecoverableSqlpools_List",
```

Comment:
>WorkspaceManagedSqlServerRecoverableSqlpools_List [](start = 24, length = 49)

Please change to `WorkspaceManagedSqlServerRecoverableSqlPools_List` to make it consistent with other operation IDs